models/interlibrary_loan_requests.rb

```diff
@@ -28,26 +28,31 @@ def self.for(uniqname:, client: ILLiadClient.new)
 class InterlibraryLoanRequest
   def initialize(parsed_response)
     @parsed_response = parsed_response
+    @title = @parsed_response["PhotoArticleTitle"] ||
+             @parsed_response["PhotoJournalTitle"]
+    @author = @parsed_response["PhotoItemAuthor"] || 
+              @parsed_response["PhotoArticleAuthor"] || 
+              @parsed_response["PhotoJournalAuthor"]
   end
   def title
-    extra = 120 - @parsed_response["LoanAuthor"].length
+    extra = 120 - @author.length
     extra = 0 if extra < 0
     max_length = 120 + extra
-    @parsed_response["LoanTitle"][0, max_length]
+    @title[0, max_length]
   end
   def author
-    extra = 120 - @parsed_response["LoanTitle"].length
+    extra = 120 - @title.length
     extra = 0 if extra < 0
     max_length = 120 + extra
-    @parsed_response["LoanAuthor"][0, max_length]
+    @author[0, max_length]
   end
```

Comment:
This should be refactored to inherit from `Item`. I think changing `Item` to have `@author` and `@title` instance variables will go a long way.

We could also try an inheritance setup like `Item`, `AlmaItem < Item`, `Loan < AlmaItem`, `Request < AlmaItem`, `InterlibraryLoanItem < Item`, `InterlibraryLoanRequest < InterlibraryLoanItem`

I can put in some of these changes since they're Alma related.

Reply:
I did wonder how `Item` could be used for `InterlibraryLoanRequest` since they're very similar. Should we create a separate branch for this task?

Reply:
Eh, I'm just working on it in here. I'll push it in soon.

Reply:
And... done.